config.py

```diff
@@ -467,22 +467,39 @@
 
 #  `event_id`` : dict
 #    Dictionary that maps events (trigger/marker values)
-#    to conditions.
+#    to conditions. The ``/``` character can be used to group conditions.
```

Comment:
```suggestion
#    to conditions. The ``/`` character can be used to group conditions.
```

Reply:
Thanks, not sure if it's my keyboard layout or VS Code constantly trying to sneak in additional backticks! super annoying